### PR TITLE
All other small fixes

### PR DESCRIPTION
**Pull request by HonkingGoose**
Here you go! The rest of the small fixes into one pull request. 🚀 

In general the changes are:

- Rewrites of some sentences for clarity
- Expand abbreviations like b/w, esp
- Add unit of measurements
- Add degrees symbol before the Celsius unit
- Fix some white-space errors
- Add punctuation where necessary
- Fix some small style annoyances (text was just too small to read comfortably)
- Fix capitalization
- Fix grammar

I've contained each change with a commit, so we can chuck out what you don't want easily. 😄 

If you want to merge the work but don't want to end with 40+ commits on the `master` branch after the merge: click on the arrow that's just next to the merge button, and select the "Squash and merge" option. That will squash all commits into a single commit. This way you'll have only one commit for all the work. See the GitHub docs on [merging ](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-a-pull-request#merging-a-pull-request-on-github) to see where to find the button.

Or you can tell me to squash all the commits, and I'll squash the lot for you myself.

Oh and can you do a linting run after merging all my changes? It will probably need to be run again, as I'm making the changes just in the text, instead of getting the full environment up and running... 🕷️ 🔫 🐛

### CODE REVIEW

File: src/app/views/cheat-sheets/basic-power/basic-power.component.html [hunk 1]
```diff
@@ -42,19 +42,19 @@ <h4 class="card-title" id="steam-power">Steam Power</h4>
...
             An <a href="https://wiki.factorio.com/Offshore_pump" target="_blank" rel="noopener">offshore pump</a>
-            provides {{ sheetData?.offshore_pump_water }} water per second.
+            provides {{ sheetData?.offshore_pump_water }} units of water per second.
         </li>
         <li>
             A <a href="https://wiki.factorio.com/Boiler" target="_blank" rel="noopener">steam boiler</a>
-            can turn up to {{ sheetData?.steam_boiler_fluid }} water units to steam per second,
-            providing {{ sheetData?.steam_boiler_energy }} of steam (at 165C).
+            can convert up to {{ sheetData?.steam_boiler_fluid }} units of water into steam per second,
+            providing {{ sheetData?.steam_boiler_energy }} of steam (at 165 °C).
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/basic-power/basic-power.component.html:50]: °C - Fancy :p

Comment (HonkingGoose) [reply src/app/views/cheat-sheets/basic-power/basic-power.component.html:50]: Some part of the guide did already use °C though... I don't remember where that was.
So now we're consistent the whole way through.

File: src/app/views/cheat-sheets/oil-refining/oil-refining.component.html [hunk 2]
```diff
@@ -27,7 +27,7 @@ <h4 id="basic-oil">Basic Oil</h4>
 
     <p>
         When starting oil, it is beneficial to rush <a href="https://wiki.factorio.com/Advanced_oil_processing_(research)" target="_blank" rel="noopener">Advanced Oil Processing</a> research.
-        <br>This unlocks Light/Heavy Oil, and allows to obtain more efficient oil ratios by converting the Light and Heavy Oil into Petroleum Gas.
+        <br>This unlocks Light/Heavy Oil, and enables more efficient oil ratios by converting the Light and Heavy Oil into Petroleum Gas.
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/oil-refining/oil-refining.component.html:30]: one omore thing, dont think i use ratios appropriately here, "processes" maybe better
```suggestion
        <br>This unlocks Light/Heavy Oil, and enables more efficient oil processes by converting the Light and Heavy Oil into Petroleum Gas.
```

File: src/app/views/cheat-sheets/science/science.component.html [hunk 3]
```diff
@@ -118,7 +118,7 @@
         <div class="col-12 col-lg-5">
             <div class="eq-container">
                 <div class="eq-field bg-factorio-green">
-                    <span class="eq-label">Research cycle time</span>
+                    <span class="eq-label">Research cycle time (seconds)</span>
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/science/science.component.html:121]: Need to keep these short, maybe just "(s)" or "in s" or get rid of time
```suggestion
                    <span class="eq-label">Research cycle seconds</span>
```

File: src/app/views/cheat-sheets/science/science.component.html [hunk 4]
```diff
@@ -135,9 +135,9 @@
     </div>
     <p class="text-muted text-center">Use the formula above to calculate how many labs satisfy a target consumption rate of science packs per minute.</p>
     <p class="text-muted">
-        Research Cycle Time = Time in seconds to complete one research cycle (usually 60 seconds for all infinite tech)<br>
+        Research Cycle Time = Time in seconds to complete one research cycle, this is usually 60 seconds for all infinite tech.<br>
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/science/science.component.html:138]: lets make the definitions bold too i think
```suggestion
        <b>Research Cycle Time</b> - Time in seconds to complete one research cycle, this is usually 60 seconds for all infinite tech.<br>
```

File: src/app/views/cheat-sheets/train-colors/train-colors.component.html [hunk 5]
```diff
@@ -3,11 +3,11 @@
     <table class="table table-sm table-hover fixed-width">
         <caption class="text-center">
             <app-blueprint-clipboard [url]="sheetData?.blueprint"></app-blueprint-clipboard>
-            Suggested locomotive colors based on cargo
+            Locomotive colors are based on the color of the cargo.
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/train-colors/train-colors.component.html:6]: This is a table title, i think its fine as it is.

File: src/app/views/cheat-sheets/vehicle-fuel-bonus/vehicle-fuel-bonus.component.html [hunk 6]
```diff
@@ -2,13 +2,13 @@
 
     <ul>
         <li>
-            <a href="https://wiki.factorio.com/Fuel" target="_blank" rel="noopener">Fuel</a> affects train max speed and
-            how fast trains accelerate.</li>
+            Use better <a href="https://wiki.factorio.com/Fuel" target="_blank" rel="noopener">Fuel</a> to increase the maximum speed and
```
Comment (deniszholob) [inline src/app/views/cheat-sheets/vehicle-fuel-bonus/vehicle-fuel-bonus.component.html:5]: If using this wording the next bullet also needs to change
"Research braking force to improve how fast the trains can slow down" or something similar.
I dont think this is needed.

File: src/app/shared/app-settings.ts [hunk 7]
```diff
@@ -15,7 +15,7 @@ export const APP_SETTINGS: AppInfo = {
         name: 'Github Source',
         url: 'https://github.com/deniszholob/factorio-cheat-sheet',
     },
-    data_version: '1.0',
+    data_version: '1.0.0',
```
[the remark this answers is not in the document]
Comment (HonkingGoose) [reply src/app/shared/app-settings.ts]: I'll leave it be, it's dropped from the pull request now.

File: src/app/views/cheat-sheets/train-colors/train-colors.component.html (hunk 5 above)
Comment (HonkingGoose) [reply src/app/views/cheat-sheets/train-colors/train-colors.component.html:6]: It's gone now.

File: src/app/views/cheat-sheets/vehicle-fuel-bonus/vehicle-fuel-bonus.component.html (hunk 6 above)
Comment (HonkingGoose) [reply src/app/views/cheat-sheets/vehicle-fuel-bonus/vehicle-fuel-bonus.component.html]: This change is also dropped.

File: src/app/views/cheat-sheets/oil-refining/oil-refining.component.html (hunk 2 above)
Comment (HonkingGoose) [reply src/app/views/cheat-sheets/oil-refining/oil-refining.component.html]: Good catch. 😄  I've committed your suggestion.

**Comment by HonkingGoose** [comment]
I've resolved your feedback, please check again if you like the end result.

**Pull request merged**